packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export interface IStyledPanel {
+  isBare?: boolean;
+  isCompact?: boolean;
+  isExpanded?: boolean;
+}
+
+const COMPONENT_ID = 'accordions.panel';
+
+const paddingStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { base } = props.theme.space;
+  let paddingTop = base * 2;
+  let paddingHorizontal = base * 5;
+  let paddingBottom = base * 8;
+
+  if (props.isCompact) {
+    paddingTop = base * 2;
+    paddingHorizontal = base * 3;
+    paddingBottom = base * 4;
+  }
+
+  if (props.isExpanded === false) {
+    paddingTop = 0;
+    paddingBottom = 0;
+  }
+
+  return css`
+    transition: padding 0.25s ease-in-out;
+    padding: ${paddingTop}px ${paddingHorizontal}px ${paddingBottom}px;
+  `;
+};
+
+export const StyledPanel = styled.div.attrs<IStyledPanel>({
```

Comment:
Prefer using semantic `section` element here and knocking out the unnecessary `role="region"` attribute in the associated `Accordion.Panel` component. See the https://garden.zendesk.com/react-containers/storybook/?path=/story/accordion-container--useaccordion story for details.

Reply:
Why was this resolved?

Reply:
This was resolved because I had made the change on my local machine and was using these comments as a checklist. I was queuing the changes up for a single push to reduce CI builds, but I can keep a separate checklist and hold off on resolving these until the commit is pushed.

Reply:
I pushed up the commit this morning and marked this thread resolved. Feel free to re-open the thread if the `Accordion.Panel` needs further adjustments.